.ci.yaml

```diff
@@ -3366,7 +3366,6 @@ targets:
     scheduler: luci
 
   - name: Mac_ios platform_view_ios__start_up
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/92422
```

Comment:
Why didn't the bot remove the flaky mark?

Reply:
Deflake bot uses prod data, and we need to update to use staging.